BetterPasswordPlugin.inc.php

```diff
@@ -208,17 +199,109 @@ function checkPassword($hookName, $args) {
 			}
 		}
 	}
-
+        /**
+	 * Check for newly added blacklists. If ran the first time creates the list of blacklists
+	 * @return 0|1|2 0 if update failed, 1 if setting is updated, 2 if new setting has been created
+	 */
+        
+        function blacklistSetting() {
+            $prevBlacklist = $this->getSetting(CONTEXT_SITE, 'betterPasswordBlacklistFiles');
+            $updateSettings = false;
+            $newBlacklist = array();
+            foreach ($this->getBlacklists() as $filename) {
+                    $newBlacklist[$filename] = sha1_file($filename);
+            }
+            if (is_null($prevBlacklist)) {
+                $this->updateSetting(CONTEXT_SITE, 'betterPasswordBlacklistFiles', $newBlacklist);
+                $updateSettings = true;
+            }
+            else {
```

Comment:
Please bind the `else`s to the closing brace of the if, to follow existing style.